Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,101 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeSourceFile
+
+Copy file to actual dir from specified path or download file
+from repository to actual dir.
+
+    limeSourceFile
+
+=over
+
+=item --source
+
+Copy file from source files to specific dir.
+
+=item --install
+
+Copy file from installed files to specific dir.
+
+=back
+
+Returns 0 when the when copy or download of file to actual dir is succesfull.
+
+=cut
+
+limeSourceFile(){
+
+    local OPTION=$1
+    local NAME=$2
+    local COPYPATH=$3
+
+    if [ -n "${COPYPATH}" ]; then
+        COPYPATH=.
+    fi
+
+    local FILEPATH=$(limeGetFilePath ${OPTION} ${NAME})
+    # get filepath to the variable and if file exist, copy file to specified dir
+    if [ -n "${FILEPATH}" ]; then
+        cp ${FILEPATH} ${COPYPATH}
+    else 
+        curl -O https://raw.githubusercontent.com/keylime/keylime/master/${NAME} ${COPYPATH}
+    fi
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeGetFilePath
+
+Trying to find path of keylime script file in several directories on the system.
+Filepath is then use for copy function, where is copied file to actual dir or
+is downloaded from repository.
+
+    limeGetFilePath
+
+=over
+
+=item --source
+
+Find the path to the specified filename in the location of the keylime source files.
+
+=item --install
+
+Find the path to the specified filename in the location of the keylime installed files.
+
+=back
+
+Returns 0 when the when getting path of file is succesfull.
```

Comment:
Rather prints found filepath to STDOUT.
Btw, current implementation won't return non-zero exit code if file is not found. If you want such a behavior, you need to change the code accordingly.